.evergreen/generate_evergreen_tasks.js

```diff
@@ -242,6 +242,44 @@ for (const compressor of ['zstd', 'snappy']) {
   });
 }
 
+// Add task for testing lambda example without aws auth.
+TASKS.push({
+  name: 'test-lambda-example',
+  tags: ['latest', 'lambda'],
+  commands: [
+    { func: 'install dependencies' },
+    {
+      func: 'bootstrap mongo-orchestration',
+      vars: {
+        VERSION: 'rapid',
```

Comment:
I don't see `test-lambda-example` in the evergreen run for this PR.  I didn't investigate fully but I think it may be related to using `rapid` here and https://github.com/mongodb/node-mongodb-native/blob/034634165cf731498613d5d5474387125df16774/.evergreen/generate_evergreen_tasks.js#L372

Reply:
Running now: https://spruce.mongodb.com/version/630e1cc03066156e2ff1d7fa/tasks?page=0&sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC&variant=%5Eubuntu1804-test-lambda%24